sssd_test_framework/utils/adcli.py

```diff
@@ -30,7 +30,7 @@ def test_adcli_join(client: Client, provider: GenericADProvider):
        kerberos based authentication.
     """
 
-    def info(self, domain: str, *, args: list[str] | None = None) -> ProcessResult:
+    def info(self, *, domain: str, args: list[str] | None = None) -> ProcessResult:
```

Comment:
You might want to have the verbose configurable to reduce the output (noise) in the future and possibly to test verbosity related bugs.

```suggestion
    def info(self, *, domain: str, verbose: bool=True, args: list[str] | None = None) -> ProcessResult:
```